### PR TITLE
MP1 Comments

### PR DESCRIPTION
**Pull request by VaibhavBajaj**
So far, I have changed the formatting in various places to better convey the meaning of the instructions.
I have also revised the rubric.

Comments:
1. Possibly remove keyboard input entirely from MP1. There seems to be no need for it. If this is done, we must make sure to change the Quit Key and Light Switch explanations where the images use keyboard keys.
2. Move Text to after Scripts so the user does not have to backtrack for the control scheme.
3. Consider adding documentation for onTriggerEnter() in Trigger Zone along with the provided manual. https://docs.unity3d.com/2018.2/Documentation/ScriptReference/Collider.OnTriggerEnter.html
4. Possibly change the layout of the webpage so that pictures are shown alongside text instead of underneath text.

These comments have not been implemented yet. They are just things to be considered.

### CODE REVIEW

File: _assignments/mp1.md [hunk 1]
```diff
@@ -12,35 +12,35 @@ rubric:
   -
     name: The Room
     points: 10
-    description: Complete and enclosed room
+    description: Complete and enclosed room.
```
Comment (andrewipark) [inline _assignments/mp1.md]: [nit] If we're going to add periods here, we should also make these into complete sentences. This applies to a few others here.

File: _assignments/mp1.md [hunk 2]
```diff
@@ -12,35 +12,35 @@ rubric:
...
   - 
     name: Text
     points: 10
-    description: Text is sharp and displays NetIDs and controls
+    description: Text is sharp and displays NetIDs and controls.
   -
     name: Skybox
     points: 10
-    description: A skybox is present from the external vantage point
+    description: A skybox is present viewable from the external vantage point.
```
Comment (andrewipark) [inline _assignments/mp1.md]:
```suggestion
    description: A skybox is viewable from the external vantage point.
```

File: _assignments/mp1.md [hunk 3]
```diff
@@ -52,23 +52,23 @@ rubric:
   -
     name: Quit Key
     points: 10
-    description: A controller button exits the game
+    description: An OVR controller button exits the game.
```
Comment (andrewipark) [inline _assignments/mp1.md]: Did you mean touch controllers here?

File: _assignments/mp1.md [hunk 4]
```diff
@@ -176,8 +176,8 @@ The inspector tab should have a `Light` component like this:
 
 ![Light Object](/img/assignments/mp1/image6.png)
 
-Of primary importance are the range (the radius of your light), color, and intensity values.
-Set the shadow type to "soft shadows", and the mode to "Realtime".
+Of primary importance are the `Range` (the radius of your light), `Color`, and `Intensity` values.
+Set the shadow type to `Soft Shadows`, and the mode to `Realtime`.
```
Comment (andrewipark) [inline _assignments/mp1.md]: [nit] I don't like this stylistically because these are UI strings, not code objects

File: _assignments/mp1.md [hunk 5]
```diff
@@ -213,11 +213,12 @@ This will create a rectangular canvas parent object with a child text object. Th
 
 First, change the Render mode from `Screen Space - Overlay`, to `World Space`.
 This changes the canvas from a UI element glued to the camera, to an object that is stationary in the world.
-Traditional UIs do not work well in VR, and we strongly discourage sticking any UI elements to the camera in your future MPs and Projects.
+
+**Note:** Traditional UIs do not work well in VR, and we strongly discourage sticking any UI elements to the camera in your future MPs and Projects.
```
Comment (andrewipark) [inline _assignments/mp1.md]: note to self: find where it says this in the oculus best practices guide

File: _assignments/mp1.md [hunk 6]
```diff
@@ -302,22 +305,22 @@ Unity's [Scripting API Reference](https://docs.unity3d.com/ScriptReference/) is
...
+#### Side Note: Input in VR
 
 Keyboards do not work well in VR, since the VR environment means that you cannot see the keyboard, and all of the keys largely feel the same.
 Controllers, with their contours and designated button shapes, are much easier to use blind.
 Refer to the following pages for more information, which you will need when you write the scripts below:
 
 - [Unity Input](https://docs.unity3d.com/Manual/ConventionalGameInput.html)
-- [Unity wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)
-- OVR Input: [general explanation](https://developer.oculus.com/documentation/unity/latest/concepts/unity-ovrinput),
+- [Unity Wiki page on Xbox controllers](http://wiki.unity3d.com/index.php?title=Xbox360Controller)
```
Comment (andrewipark) [inline _assignments/mp1.md]: Can we remove this since we're not using XBox controllers?

File: _assignments/mp1.md [hunk 7]
```diff
@@ -372,9 +375,9 @@ To change the light, write something similar to the following, but remember to u
 ![Get Tab Key Down](/img/assignments/mp1/image14.png)
 
 For the actual light color, you can either create a new color using the [`new Color(red, green, blue)`](https://docs.unity3d.com/ScriptReference/Color-ctor.html) constructor, or one of the [predefined colors](https://docs.unity3d.com/ScriptReference/Color.html).
-How you change the light is up to you, but pressing the controller button you designate must visibly change the light color.
+How you change the light is up to you, but pressing the light switch controller button repeatedly must visibly change the light color each time.
```
Comment (andrewipark) [inline _assignments/mp1.md]: This feels really syntactically strange.

File: _assignments/mp1.md [hunk 8]
```diff
@@ -384,28 +387,27 @@ Since the moon object is a child of the planet object, it will also rotate aroun
...
 > Set up a button on the controllers to switch the player's position between an external viewing point and the room.
 
 Like all `GameObject`s, the player object has a `Transform`.
 With that in mind, make a script that switches the player between a room and a new external viewing point,
 e.g. a small new plane a moderate distance from the room.
-The player should still start within the room.
-Pressing the button for the first time must move the player to the external viewing point.
-After that, pressing the button must alternate between the two locations.
+
+The player should  start within the room. Pressing the button for the first time must move the player to the external viewing point. After that, pressing the button must alternate between the two locations.
```
Comment (andrewipark) [inline _assignments/mp1.md]:
```suggestion
The player should start within the room. Pressing the button for the first time must move the player to the external viewing point. After that, pressing the button must alternate between the two locations.
```

File: _assignments/mp1.md (hunk 3 above)
Comment (VaibhavBajaj) [reply _assignments/mp1.md]: Yes. That or Oculus controller.

File: _assignments/mp1.md (hunk 4 above)
Comment (VaibhavBajaj) [reply _assignments/mp1.md]: The only reason I did this was because it felt inconsistent throughout the document.

File: _assignments/mp1.md (hunk 6 above)
Comment (VaibhavBajaj) [reply _assignments/mp1.md]: Sure

File: _assignments/mp1.md (hunk 7 above)
Comment (VaibhavBajaj) [reply _assignments/mp1.md]: I can rephrase it. All I want to communicate is that repeatedly pressing the button should repeatedly change the color.

**Comment by andrewipark** [comment]
leaving this PR open for further revision to mp1 prose.

**Pull request closed without merging**